src/Select/SelectInput.js

```diff
@@ -251,9 +251,11 @@ class SelectInput extends React.Component {
       }
 
       return React.cloneElement(child, {
+        onClick: this.handleItemClick(child),
         role: 'option',
         selected,
-        onClick: this.handleItemClick(child),
+        value: undefined, // The value is most likely not a valid HTML attribute.
```

Comment:
hi @oliviertassinari thank you for the PR.

I have a little question on this change: why do we need it? Asking because I was debugging like a day an issue caused by this change. Here is reproducible example: https://codesandbox.io/s/y3n9lv7lpz
What I basically do:
instead of having all my components in one file, I import menu item wrapper from other place. As soon as I have a wrapper for this `MenuItem`, it doesn't work anymore as `value` prop is always undefined. I surely can get it from 'data-` attribute but as this PR doesn't contain any docs or tests I'd be welcome to hear your thoughts first rather than go and add docs/tests.
Thank you.

Reply:
@goodniceweb Yes, it's expected. You need to forward the properties injected. I know, it's implicit :/. It's a bit documented in https://material-ui-next.com/guides/composition/#wrapping-components.

```jsx
const Item = function({ label, ...other }) {
  return <MenuItem {...other}>{label}</MenuItem>;
};
```